optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/tenant/TenantRestServiceImpl.java

```diff
@@ -55,9 +55,12 @@ public Tenant addTenant(Tenant tenant) {
 
     @Override
     @Transactional
-    public Tenant updateTenantConfiguration(TenantConfigurationView tenantConfigurationView) {
-        Tenant tenant = entityManager.find(Tenant.class, tenantConfigurationView.getTenantId());
-        tenant.getConfiguration().setView(tenantConfigurationView);
+    public Tenant updateTenantConfiguration(TenantConfiguration tenantConfiguration) {
+        Tenant tenant = entityManager.find(Tenant.class, tenantConfiguration.getTenantId());
+        if (null == tenant) {
+        	throw new IllegalStateException("There is no tenant with id " + tenantConfiguration.getTenantId());
```

Comment:
identation

Reply:
For exception messages consistency, please follow this style:
- The exception message should be a sentance, so it should end with punctuation (normally a ".", avoid the use of "!").
- Mention as many variable as possible+relevant and mention them with this pattern: " variableName (" + variable + ")"

So this error message becomes:

`throw new IllegalStateException("There is no tenant with id (" + tenantConfiguration.getTenantId() + ").");`